### PR TITLE
Implement Tx Details From/To rows

### PR DESCRIPTION
**Pull request by jkadamczyk**
Fixes APP-31

## What changed (plus any additional context for devs)

* Implemented a new From / To section for the transaction details sheet
* Added press handler for transaction hash row. Now pressing copies the transaction hash to clipboard

### CODE REVIEW

File: src/languages/_english.json [hunk 1]
```diff
@@ -1428,9 +1430,13 @@
       }
     },
     "transaction_details": {
+      "from": "From",
+      "to": "To",
       "value": "Value",
       "hash": "Tx Hash",
-      "network_fee": "Network Fee"
+      "network_fee": "Network Fee",
+      "hash_copied": "\uDBC0\uDC63 Transaction Hash Copied",
```
Comment (jinchung) [inline src/languages/_english.json:1438]: nit - just wanted to confirm `hash_copied` and `address_copied` are supposed to have that capitalization on each word (vs just the first word, which seems to be the pattern for our toasts content). Didn't see this defined on the figma anywhere.

File: src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx [hunk 2]
```diff
@@ -22,27 +23,32 @@ export const TransactionDetailsValueAndFeeSection: React.FC<Props> = ({
...
+    <>
+      <TransactionDetailsDivider />
+      <Box paddingVertical="20px">
+        <Stack space="20px">
+          {value && (
+            <DoubleLineTransactionDetailsRow
+              leftComponent={
+                <CoinIcon address={coinAddress} symbol={coinSymbol} />
+              }
+              title={i18n.t(i18n.l.transaction_details.value)}
```
Comment (jinchung) [inline src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx:35]: will this be the new pattern going forward?

File: src/screens/transaction-details/TransactionDetails.tsx [hunk 3]
```diff
@@ -61,13 +87,20 @@ export const TransactionDetails: React.FC<Props> = ({ navigation, route }) => {
           backgroundColor={backgroundColor}
           height={IS_ANDROID ? sheetHeight : '100%'}
           deferredHeight={IS_ANDROID}
+          scrollEnaled={false}
```
Comment (jinchung) [inline src/screens/transaction-details/TransactionDetails.tsx:90]: typo `scrollEnabled`

**Review by skylarbarrera — APPROVED** [review]
lgtm after wins comments

File: src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx (hunk 2 above)
Comment (skylarbarrera) [reply src/screens/transaction-details/components/TransactionDetailsValueAndFeeSection.tsx:35]: yesh

File: src/hooks/useWalletBalances.ts [hunk 4]
```diff
@@ -14,11 +13,13 @@ import { queryClient } from '@/react-query';
...
 import { fromWei, handleSignificantDecimals } from '@/helpers/utilities';
 import logger from '@/utils/logger';
+import { useSelector } from 'react-redux';
+import { AppState } from '@/redux/store';
 
 const ETH_ADDRESS = '0x0000000000000000000000000000000000000000';
 
 const useWalletBalances = (wallets: AllRainbowWallets) => {
-  const { network } = useAccountSettings();
+  const network = useSelector((state: AppState) => state.settings.network);
```
Comment (estrattonbailey) [inline src/hooks/useWalletBalances.ts:22]: Nice!

**Review by estrattonbailey — APPROVED** [review]
Yep LGTM too

**Pull request merged**